settings.php

```diff
@@ -31,7 +31,16 @@
     $settings->add(new admin_setting_heading(
         'enrol_oneroster',
         '',
-        get_string('pluginname_desc', 'enrol_database')
+        get_string('options', 'enrol_oneroster')
+    ));
+
+    // CSV upload link.
+    $settings->add(new admin_setting_heading(
```

Comment:
I dont think you still need the link inline. Setting a new external page adds it to the nav tree directly